src/app/controllers/SurveyController.js

```diff
@@ -31,6 +30,31 @@ class SurveyController {
         });
     }
 
+    /**
+     * Update a Survey and return the ID of the updated object if successful
+     * @param {string} id - The id of the survey to be modified
+     * @param {Object} survey - The Modified Survey Object
+     * @returns {Promise<any>}
+     */
+    update(id, survey){
```

Comment:
Is there some authentication for these update mutations? Like what happens if someone just tries to update everyone's survey but they aren't the vendor that owns it etc?